{"resourceLogs":[{"resource":{"attributes":[{"key":"host.name","value":{"stringValue":"f2560f55770a"}},{"key":"container.id","value":{"stringValue":"f2560f55770a7f5076f1dc81341d814b5017820700e56672d9b8a61919b266ad"}},{"key":"service.name","value":{"stringValue":"cartservice"}},{"key":"service.code","value":{"stringValue":"CA1003"}},{"key":"telemetry.sdk.name","value":{"stringValue":"opentelemetry"}},{"key":"telemetry.sdk.language","value":{"stringValue":"dotnet"}},{"key":"telemetry.sdk.version","value":{"stringValue":"1.9.0"}}]},"scopeLogs":[{"scope":{"name":"Grpc.AspNetCore.Server.ServerCallHandler"},"logRecords":[{"timeUnixNano":"1726756836723459900","observedTimeUnixNano":"1726756836723459900","severityNumber":17,"severityText":"Error","body":{"stringValue":"Error when executing service method '{ServiceMethod}'."},"attributes":[{"key":"exception.type","value":{"stringValue":"IOException"}},{"key":"exception.message","value":{"stringValue":"The request stream was aborted."}},{"key":"exception.stacktrace","value":{"stringValue":"System.IO.IOException: The request stream was aborted.\n ---\u003e Microsoft.AspNetCore.Connections.ConnectionAbortedException: The HTTP/2 connection faulted.\n --- End of inner exception stack trace ---\n at System.IO.Pipelines.Pipe.GetReadResult(ReadResult\u0026 result)\n at System.IO.Pipelines.Pipe.GetReadAsyncResult()\n at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http2.Http2MessageBody.ReadAsync(CancellationToken cancellationToken)\n at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource\u003cTResult\u003e.GetResult(Int16 token)\n at Grpc.AspNetCore.Server.Internal.PipeExtensions.ReadSingleMessageAsync[T](PipeReader input, HttpContextServerCallContext serverCallContext, Func`2 deserializer)\n at Grpc.AspNetCore.Server.Internal.CallHandlers.UnaryServerCallHandler`3.HandleCallAsyncCore(HttpContext httpContext, HttpContextServerCallContext serverCallContext)\n at Grpc.AspNetCore.Server.Internal.CallHandlers.ServerCallHandlerBase`3.\u003cHandleCallAsync\u003eg__AwaitHandleCall|8_0(HttpContextServerCallContext serverCallContext, Method`2 method, Task handleCall)"}},{"key":"ServiceMethod","value":{"stringValue":"GetCart"}}],"flags":1,"traceId":"d08a853b422dd7bfef8a2849cf9d732d","spanId":"fe5013dfe7579858"},{"timeUnixNano":"1726756836723459900","observedTimeUnixNano":"1726756836723459900","severityNumber":17,"severityText":"Error","body":{"stringValue":"Error when executing service method '{ServiceMethod}'."},"attributes":[{"key":"exception.type","value":{"stringValue":"IOException"}},{"key":"exception.message","value":{"stringValue":"The request stream was aborted."}},{"key":"exception.stacktrace","value":{"stringValue":"System.IO.IOException: The request stream was aborted.\n ---\u003e Microsoft.AspNetCore.Connections.ConnectionAbortedException: The HTTP/2 connection faulted.\n --- End of inner exception stack trace ---\n at System.IO.Pipelines.Pipe.GetReadResult(ReadResult\u0026 result)\n at System.IO.Pipelines.Pipe.GetReadAsyncResult()\n at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http2.Http2MessageBody.ReadAsync(CancellationToken cancellationToken)\n at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource\u003cTResult\u003e.GetResult(Int16 token)\n at Grpc.AspNetCore.Server.Internal.PipeExtensions.ReadSingleMessageAsync[T](PipeReader input, HttpContextServerCallContext serverCallContext, Func`2 deserializer)\n at Grpc.AspNetCore.Server.Internal.CallHandlers.UnaryServerCallHandler`3.HandleCallAsyncCore(HttpContext httpContext, HttpContextServerCallContext serverCallContext)\n at Grpc.AspNetCore.Server.Internal.CallHandlers.ServerCallHandlerBase`3.\u003cHandleCallAsync\u003eg__AwaitHandleCall|8_0(HttpContextServerCallContext serverCallContext, Method`2 method, Task handleCall)"}},{"key":"ServiceMethod","value":{"stringValue":"GetCart"}}],"flags":1,"traceId":"3c7829d88f923d64d55339655779a249","spanId":"b51747c163a1cd63"}]}]}]}
{"resourceLogs":[{"resource":{"attributes":[{"key":"container.id","value":{"stringValue":"3664da43153e5051d2c104b66787eed90be022efd5f3d45eec9b9a4e47b4c718"}},{"key":"docker.cli.cobra.command_path","value":{"stringValue":"docker compose"}},{"key":"host.arch","value":{"stringValue":"amd64"}},{"key":"host.name","value":{"stringValue":"3664da43153e"}},{"key":"os.description","value":{"stringValue":"Linux 5.15.153.1-microsoft-standard-WSL2"}},{"key":"os.type","value":{"stringValue":"linux"}},{"key":"process.command_line","value":{"stringValue":"/opt/java/openjdk/bin/java -javaagent:/usr/src/app/opentelemetry-javaagent.jar oteldemo.AdService"}},{"key":"process.executable.path","value":{"stringValue":"/opt/java/openjdk/bin/java"}},{"key":"process.pid","value":{"intValue":"1"}},{"key":"process.runtime.description","value":{"stringValue":"Eclipse Adoptium OpenJDK 64-Bit Server VM 21.0.4+7-LTS"}},{"key":"process.runtime.name","value":{"stringValue":"OpenJDK Runtime Environment"}},{"key":"process.runtime.version","value":{"stringValue":"21.0.4+7-LTS"}},{"key":"service.instance.id","value":{"stringValue":"31c865c7-18df-46f7-81ea-1e4f48bc4201"}},{"key":"service.name","value":{"stringValue":"adservice"}},{"key":"telemetry.distro.name","value":{"stringValue":"opentelemetry-java-instrumentation"}},{"key":"telemetry.distro.version","value":{"stringValue":"2.6.0"}},{"key":"telemetry.sdk.language","value":{"stringValue":"java"}},{"key":"telemetry.sdk.name","value":{"stringValue":"opentelemetry"}},{"key":"telemetry.sdk.version","value":{"stringValue":"1.40.0"}}]},"scopeLogs":[{"scope":{"name":"oteldemo.AdService"},"logRecords":[{"timeUnixNano":"1724247958160694394","observedTimeUnixNano":"1724247958160701884","severityNumber":13,"severityText":"WARN","body":{"stringValue":"GetAds Failed with status Status{code=UNAVAILABLE, description=null, cause=null}"},"flags":1,"traceId":"ed8b4191838daec751003df550476f1d","spanId":"21154ad78808c30e"}]}],"schemaUrl":"https://opentelemetry.io/schemas/1.24.0"}]}